sdk/openai/openai/src/models/models.ts

```diff
@@ -79,7 +79,10 @@ export interface PromptFilterResult {
 }
 
 /** Information about the content filtering category, if it has been detected. */
-export interface ContentFilterResults {
+export type ContentFilterResults = ContentFilterSuccessOutput | ContentFilterErrorOutput;
```

Comment:
Why do we have this change in the model under `models/models.ts`? I expect it to be only under `api/models.ts`.